CONTRIBUTE.md

```diff
@@ -0,0 +1,17 @@
+The following is based off of the [Rails Contribution guidelines][].
+
+Spacemacs is a volunteer effort. We encourage you to pitch in. The community makes Spacemacs what it is. We have a few guidelines, which we ask all contributors to follow.
+
+- If you want to ask a usage question, be sure to read [our documentation][], as it may hold the answer. You should also reference the `README.org` of the relevant layer(s). If your question is not answered there, then feel free to ask on our [gitter chat][].
+- If you want to submit a PR, please read [our contribution guidelines][] before submitting. Also, make sure that your Pull Request (PR) is against the `develop` branch and, not `master`.
+- If you are opening an issue, consider seeing if the bug has been fixed on `develop` and, that you are up to date by doing `git pull --rebase`. If it is not fixed, please include the output of `SPC h d s` in your bug report. This will include the following OS: Emacs Version, Spacemacs version, Spacemacs branch(`master` or `develop`), as well as current HEAD revision, distribution (`spacemacs` or `spacemacs-base`) and what layers you are using. This will help us to help you better.
```

Comment:
The colon should be after "following", not after "OS". Version is capitalized in one place but not in another. Also add a space after "branch". :-)